src/main/java/com/github/igotyou/FactoryMod/ConfigParser.java

```diff
@@ -810,6 +814,25 @@ private IRecipe parseRecipe(ConfigurationSection config) {
 		case "DUMMY":
 			result = new DummyParsingRecipe(identifier, name, productionTime, null);
 			break;
+		case "PRINTINGPLATE":
+			ItemMap printingPlateInput = parseItemMap(config.getConfigurationSection("input"));
```

Comment:
You dont need to parse the input again here (same for the other recipes), because pretty much all the recipes use some input it is already parsed out before this switch and you can use it with the "input" variable.


Additionally it would be good if those new recipes supported config inheritation the way the other recipes do to keep things consistent.